target/linux/ipq40xx/image/ipq-aux-loader/src/Makefile

```diff
@@ -0,0 +1,162 @@
+#
+# Auxiliary kernel loader for Qualcom IPQ-4XXX/806X based boards
+#
+#  Copyright (C) 2019 Sergey Sergeev <adron@mstnt.com>
+#
+#  This program is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU General Public License version 2 as published
+#  by the Free Software Foundation.
+#
+
+CPU_TYPE := IPQ4XXX
+#+72M. realloc memory address
+TEXT_BASE	:= 0x44800000
+#RouterBOOT auto realloc flag value address. Kernel size is limited < 6M !
+TEXT_BASE2 := 0x01100000
+#fat kernels start size 5M
+FAT_SIZE_START := 5000000
+#for fat kernels <= 12M
+TEXT_BASE2_FAT := 0x00000000
+
+#ubi image vars
+BLOCKSIZE := 256k
+PAGESIZE  := 4096
+
+CC      := $(CROSS_COMPILE)gcc
+LD      := $(CROSS_COMPILE)ld
+OBJCOPY	:= $(CROSS_COMPILE)objcopy
+OBJDUMP	:= $(CROSS_COMPILE)objdump
+
+BIN_FLAGS	:= -O binary -R .ARM.attributes -R .comment -R .debug.* -S
+
+CFLAGS = -D__KERNEL__ -DCONFIG_SYS_TEXT_BASE=$(TEXT_BASE) 	 			\
+			-DCONFIG_ARM -D__ARM__ -fPIC -Wall -Wstrict-prototypes 			\
+			-Wno-format-security -Wno-format-nonliteral 								\
+			-fno-stack-protector -fstack-usage -pipe 										\
+			-marm -mno-thumb-interwork -mabi=aapcs-linux -march=armv7-a \
+			-mno-unaligned-access -fno-builtin -ffreestanding 				  \
+			-g -Os -fno-common -ffixed-r8
```

Comment:
Still a lot of added spaces here.
Maybe grep your changed files with some like
grep "  \" (two spaces plus backslash)

Reply:
As with the other suggestion, maybe do git format-patch on all changes first, and then only grep your patches. This will prevent a lot of results from code you are not touching.

Reply:
Yeah, this is gonna be a pain to sort out.
Aux loader was not written by me so its gonna take some work to chase all of the excess spaces.